server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolParser.g4

```diff
@@ -998,7 +998,8 @@ externalStatement
 
 dafStatements
     : readTransactionStatement | writeTransactionStatement | writeReportStatement
-    | openPacketStatement
+    | openPacketStatement | getEntityStatement | getUserStatements | getItemStatements
```

Comment:
I would recommend extracting all the references to the `get...` rules in a separate rule. It will allow pulling the `GET` keyword to the top of the rule hierarchy to decrease the ambiguity. Also, you may do it for the `getEntity...` and `getItem...` rules with 'ENTITY` and `ITEM` keywords respectively. It is going to increase the performance drastically.